### PR TITLE
Feature/epam builder fix

### PR DESCRIPTION
**Pull request by garyluu**
Update pipeline-builder to latest dev.

Also adds a simple test to check if the library (not our code) is broken.

See failed first commit to confirm that it's broken.

See last two commit for passing tests

### CODE REVIEW

**Comment by garyluu** [comment]
> Does the upgrade have any bearing on [dockstore/dockstore#2461](https://github.com/dockstore/dockstore/issues/2461)?

Doesn't look like it.  Seems like the Angular 8 update may have fixed it though.

File: package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "ngx-bootstrap": "^3.0.1",
     "ngx-clipboard": "^12.2.1",
     "ngx-markdown": "^8.1.0",
-    "pipeline-builder": "^0.3.10-dev.298",
+    "pipeline-builder": "^0.3.10-dev.313",
```
Comment (denis-yuen) [inline package.json]: Do they have non-dev releases?
(Is this the equivalent on a SNAPSHOT in Maven?)

File: src/app/workflow/dag/wdl-viewer/state/wdl-viewer.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
+/*
+ *    Copyright 2017 OICR
```
Comment (denis-yuen) [inline src/app/workflow/dag/wdl-viewer/state/wdl-viewer.service.spec.ts]: Getting a bit dated

File: package.json (hunk 1 above)
Comment (garyluu) [reply package.json]: They do, but it hasn't been updated in forever

Comment (denis-yuen) [reply package.json]: :disappointed:

**Pull request merged**